### PR TITLE
US107607 Made notification dot a component

### PR DESCRIPTION
**Pull request by ciss1995**
US107607
https://github.com/BrightspaceUI/navigation/pull/69

### CODE REVIEW

File: d2l-navigation-notification-icon.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+class NavigationNotificationIcon extends PolymerElement {
+	static get template() {
+		return html`
+            <style is="custom-style">
+                :host {
+                    display: inline-block;
+                    height: 100%;
+                    position: absolute;
+                    right: calc(-50% - 4px);
```
Comment (ciss1995) [inline d2l-navigation-notification-icon.js]: We would need to change this calculation to allow the component to work with https://github.com/BrightspaceHypermediaComponents/organizations/blob/master/components/d2l-organization-consortium/d2l-organization-consortium-tabs.js. I tried `right: calc(-50% - 11px);`, which seems to work but I am not sure if that reflects the actual padding we want.

**Comment by edwardg8** [comment]
Looks good to me, i'm not sure about positioning of the dot.

File: d2l-navigation-notification-icon.js (hunk 1 above)
Comment (svanherk) [reply d2l-navigation-notification-icon.js]: So is the plan to make this configurable?  Or can you override the style in d2l-org-tabs as is?

File: d2l-navigation-button.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ class D2LNavigationButton extends mixinBehaviors([D2L.PolymerBehaviors.Button.Be
 	static get template() {
 		const template = html`
 				${highlightStyles}
-				<style is="custom-style">
+				<style is="custom-style" include="d2l-offscreen-shared-styles">
```
Comment (svanherk) [inline d2l-navigation-button.js]: Nice, looks like this should've always been here

File: d2l-navigation-notification-icon.js (hunk 1 above)
Comment (svanherk) [reply d2l-navigation-notification-icon.js]: I'm thinking it should be configurable as is, since it's in the `:host` selector

**Comment by ciss1995** [comment]
After discussing it with @svanherk we agreed to change `right: calc(-50% - 4px);` to `right: calc(-100% + 11px);` to allow it to work with `d2l-organization-consortium-tabs`. We will have to do some regression testing on the nav bar.

**Pull request merged**